sources/api/host-containers/src/main.rs

```diff
@@ -437,8 +437,32 @@ where
     Ok(())
 }
 
+fn is_restart_required(settings: &[&str], container_name: &str) -> bool {
+    if settings.is_empty() {
+        // it means that Bottlerocket is booting - all containers need to be started
+        return true;
+    }
+    let setting_prefix = "settings.host-containers";
```

Comment:
I don't understand the change to remove the final `.`.  thar-be-settings gets a list of concrete settings that changed, and `settings.host-containers` isn't a setting you can change on its own, it's only a container for other settings, so everything we receive should start with `settings.host-containers.` if it's related to host containers.  What am I missing?

Reply:
We are appending another dot below here for container_name` let container_prefix = format!("{}.{}.", setting_prefix, container_name);` so the container prefix was `settings.admin..enabled` earlier, because of which it was returning false. Do you want me to change container_prefix rather ?

Reply:
Yeah, I think we should remove the inner `.` from `container_prefix` so we can check `setting_prefix` correctly.